avto-bus/avto-bus.js

```diff
@@ -1,35 +1,30 @@
-function showMinimum(wheels) {
-   if(wheels % 6 === 0) {
-       console.log("Minimum number of busses is:" + wheels/6)
-   }
+// https://codeforces.com/problemset/problem/1679/A
+
+var wheels = 24;
+var busSixWheels = 6;
+var busFourWheels = 4;
+
+function checkNumber(wheels) {
+  if (wheels % busSixWheels === 0 && wheels % busFourWheels !== 0) {
+    return numDivisibleBySix(wheels);
+  } else if (wheels % busFourWheels === 0 && wheels % busSixWheels !== 0) {
+    return divisibleByFour;
+  } else if (wheels % busSixWheels === 0 && wheels % busFourWheels === 0) {
+    var numDeivisbleBySicAndFour = [divisibleBySix, divisibleByFour];
+    return numDeivisbleBySicAndFour;
+  } else if (wheels % 4 !== 0 && wheels % 6 !== 0) {
+    return -1;
+  }
 }
-
-function showMaximum(wheels) {
-    if(wheels % 4 === 0){
-        console.log("Maximum number of busses is:" + wheels/4)
-    }
-}
-
-function noSuitableNumber(wheels) {
-    if(wheels % 6 !== 0 && wheels % 4 !== 0){
-        console.log("-1")
-    }
+function numDivisibleBySix(wheels) {
```

Comment:
to raczej mialo troche inaczej wygladac w zamysle.
Aby uniknac pisania modulo w ifie. Wiec zamiast dzielic z reszta w funkcji i potem costam sprawdzac. To sprawdzaj w funkcji przez dzielenie z reszta i return true albo false zaleznie od tego czy sie dzieli czy nie

Reply:
Poprawione, patrz dalej.